### DISCUSSION TITLE
[3.0] User management --> web users HOW TO?

### DISCUSSION THREAD

**Discussion by leonie12** [evolution-cms/evolution#1909] [Q&A]
Until now I have used weblogin and personalize snippet to enable web user groups to login to see corresponding pages and navigation.
This is broken now (as was announced with RC2?). Is there anywhere an explanation/tutorial how to convert existing login(user management to new management in English and at best understandable for website-owners with limited coding knowledge? 

For me one big advantage of Modx/Evo was besides its flexibility its flat learning curve. With some basic PHP knowledge everything was solvable. This seems to have changed a bit. In combination with non-existing forums there seems to be a big need for easy-to-understandable tutorials. Are they existing anywhere unknown?

**Comment by fourroses666** [comment]
There is a Telegram group where you can ask English questions. My coding skills are also not that large and the group is very handy when needed. https://t.me/evolutioncms 

The user management is different in 3.0 but don't know if there is an tutorial available yet.

**Comment by Dmi3yy** [comment]
Soon will new version for Formlister for work with users 
Or use this docs: 

https://docs.evo.im/03_develop/06_di_v_evolution_cms/05_usermanager.html


Weblogin not supported many years 
But if need we can rewrite this solution but like a work

**Comment by leonie12** [comment]
I am voluntary responsible for the websites of two small local NGOs. There is no money for paying for development.
If weblogin is deprecated for years, what was its successor? Is there anywhere an explanation available how to convert from this to 3.0 management. Above link is far above my coding skills and certainly also above the possibilities of other long year evo users. Also my russian language knowledge is very limited ;-) .
Any help or hint in the right direction is much appreciated.

**Comment by Dmi3yy** [comment]
Need wait until formlister will work on Evo 3.0 and only then update, it will be not dificult

**Comment by scenduk** [comment]
Hi Dmi3yy, do you know when the new version of FormLister might be ready? I'm moving a couple of membership sites from WebLoginPE at the moment and was nearly finished in Evo 2. I'm happy to help pay for development if that can be prioritised - especially if it includes docs on how to have custom user fields with FormLister registration - it would be great to ultimately be able to add fields to users as easily as TVs to resources.

**Comment by Dmi3yy** [comment]
Not ready yet, add  Pathologic  hi is do that task.

**Comment by Pathologic** [comment]
One of core components is blocking FormLister from work. I'll fix it soon.

**Comment by Pathologic** [comment]
https://github.com/Pathologic/MODxAPI/blob/master/README.md

**Comment by BBloke** [comment]
I'll be heading down this route soon enough.  I successfully migratred from WebLoginPE to FormLister.  Although I had a couple of work arounds to have it stable.

I've done a test update from 2.0.4 to 3.1.1 via the migration script.  Obviously none of my own code which utilises web_groups etc works so I need to rewrite that. 

I cannot for the life of me find where to look for the stuff to work out with users and groups to change my code!

Looking forward to seeing some docs or tutorials or enough info so I can plough through it myself.

**Comment by scenduk** [comment]
Pathologic has been helping me iron out some bugs but I think I'm nearly there with Evo 3 and FormLister for login, registering, profile editing including using TVs for web user custom fields. Once I'm done I'll post some info on the forum and a link here.

**Comment by BBloke** [comment]
That would be wonderful.  It looks like it's become more abstract than my head can manage!

I can obviously turn off all my snippets in the templates for now but I'd rather know how to fix it going forward if v2 is no more and v3 will be the next stop.

**Comment by BBloke** [comment]
I found what I was looking for.  Everything from webgroups has moved to membergroups.  I can change my code accordingly when the time comes and work on logins when there is more information.

**Comment by BBloke** [comment]
I wrote a migration script to move v1 and v2 users extended attributes to template variables:

https://forum.evo.im/d/70-upgrade-user-from-v204-to-v3-tv-roles/10

It basically takes the extended table column names and makes the TV's, assigns all users to Power User Role.  Assigns TV to Power User Role.  Populates values from extended table to applicable new TV.
